receiver/kubeletstatsreceiver/README.md

```diff
@@ -187,5 +187,50 @@ The following parameters can also be specified:
 The full list of settings exposed for this receiver are documented [here](./config.go)
 with detailed sample configurations [here](./testdata/config.yaml).
 
+## Metrics
+
+Details about the metrics produced by this receiver can be found in [metadata.yaml](./metadata.yaml) with further documentation in [documentation.md](./documentation.md)
+
+### Feature gate configurations
+
+#### Transition from metrics with "direction" attribute
+
+Some kubeletstats metrics reported are transitioning from being reported with a `direction` attribute to being reported with the
+direction included in the metric name to adhere to the OpenTelemetry specification
+(https://github.com/open-telemetry/opentelemetry-specification/pull/2617):
+
+- `k8s.node.network.io` will become:
+  - `k8s.node.network.io.transmit`
+  - `k8s.node.network.io.receive`
+- `k8s.node.network.errors` will become:
+  - `k8s.node.network.errors.transmit`
+  - `k8s.node.network.errors.receive`
+
+The following feature gates control the transition process:
+
+- **receiver.kubeletstatsreceiver.emitMetricsWithoutDirectionAttribute**: controls if the new metrics without `direction` attribute are emitted by the receiver.
+- **receiver.kubeletstatsreceiver.emitMetricsWithDirectionAttribute**: controls if the deprecated metrics with `direction` attribute are emitted by the receiver.
+
+##### Transition schedule:
+
+1. v0.58.0, Aug 2022:
+
+- The new metrics are available for all scrapers, but disabled by default, they can be enabled with the feature gates.
+- The old metrics with `direction` attribute are deprecated with a warning.
+- `receiver.kubeletstatsreceiver.emitMetricsWithDirectionAttribute` is enabled by default.
+- `receiver.kubeletstatsreceiver.emitMetricsWithoutDirectionAttribute` is disabled by default.
+
+2. v0.59.0, August 2022:
+
+- The new metrics are enabled by default, deprecated metrics disabled, they can be enabled with the feature gates.
+- `receiver.kubeletstatsreceiver.emitMetricsWithDirectionAttribute` is disabled by default.
+- `receiver.kubeletstatsreceiver.emitMetricsWithoutDirectionAttribute` is enabled by default.
+
+3. v0.60.0, September 2022:
+
+- The feature gates are removed.
+- The new metrics without `direction` attribute are always emitted.
+- The deprecated metrics with `direction` attribute are no longer available.
+
```

Comment:
```suggestion
```

removing this section as agreed in https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/11815